### PR TITLE
Update logging document to fix the inactive links

### PR DESCRIPTION
**Pull request by w13915984028**
issues:

https://github.com/harvester/harvester/issues/6824

### CODE REVIEW

File: versioned_docs/version-v1.3/logging/harvester-logging.md [hunk 1]
```diff
@@ -285,7 +285,7 @@ tls_cert_path:
          key: <kubernetes-secret-key>
 ```
 
-For more information, you can find the related documentation [here](https://banzaicloud.com/docs/one-eye/logging-operator/configuration/plugins/outputs/secret/).
+For more information, you can find the related documentation [here](https://kube-logging.dev/docs/configuration/output/).
```
Comment (brandboat) [inline versioned_docs/version-v1.3/logging/harvester-logging.md:288]:
```suggestion
For more information, you can find the related documentation [here](https://kube-logging.dev/docs/configuration/plugins/outputs/secret/).
```

**Review by brandboat — COMMENTED** [review]
Thanks!

File: docs/logging/harvester-logging.md [hunk 2]
```diff
@@ -285,7 +285,7 @@ tls_cert_path:
          key: <kubernetes-secret-key>
 ```
 
-For more information, you can find the related documentation [here](https://banzaicloud.com/docs/one-eye/logging-operator/configuration/plugins/outputs/secret/).
+For more information, you can find the related documentation [here](https://kube-logging.dev/docs/configuration/output/).
```
Comment (brandboat) [inline docs/logging/harvester-logging.md:288]:
```suggestion
For more information, you can find the related documentation [here](https://kube-logging.dev/docs/configuration/plugins/outputs/secret/).
```

File: docs/logging/harvester-logging.md [hunk 3]
```diff
@@ -36,9 +36,9 @@ For Harvester clusters upgraded from version v1.1.x, the logging feature is conv
 
 ## High-level Architecture
 
-The [Banzai Cloud Logging operator](https://banzaicloud.com/docs/one-eye/logging-operator/) now powers both `Harvester` and `Rancher` as an in-house logging solution.
+The [Logging Operator](https://kube-logging.dev/docs/#overview) now powers both `Harvester` and `Rancher` as an in-house logging solution.
```
Comment (brandboat) [inline docs/logging/harvester-logging.md:39]: We need to update those links in v1.2 doc, right ?

Comment (w13915984028) [reply docs/logging/harvester-logging.md:39]: v1.2 may be EOL. Please @jillian-maroket confirm, should we update v1.2 part as well? thanks.

Comment (jillian-maroket) [reply docs/logging/harvester-logging.md:39]: According to the [Support Matrix](https://www.suse.com/suse-harvester/support-matrix/all-supported-versions/harvester-v1-2-x/), the EOL date for v1.2.x is 08 September 2024.

@bk201 Please confirm so we can finalize the scope of this PR. I'll also add the `EOL` label to v1.2.0 in the header's version selector.

**Review by brandboat — APPROVED** [review]
LGTM

File: docs/logging/harvester-logging.md [hunk 4]
```diff
@@ -169,11 +169,11 @@ Depending on the output selected (Splunk, Elasticsearch, etc), there will be add
 
 ###### Output
 
-The fields present in the **Output** form will change depending on the `Output` chosen, in order to expose the fields present for each [output plugin](https://banzaicloud.com/docs/one-eye/logging-operator/configuration/plugins/outputs/).
+The fields present in the **Output** form will change depending on the `Output` chosen, in order to expose the fields present for each [output plugin](https://kube-logging.dev/docs/configuration/plugins/outputs/).
```
Comment (jillian-maroket) [inline docs/logging/harvester-logging.md:172]:
```suggestion
The form shows the fields that are available for the selected [output](https://kube-logging.dev/docs/configuration/plugins/outputs/).
```

File: docs/logging/harvester-logging.md [hunk 5]
```diff
@@ -169,11 +169,11 @@ Depending on the output selected (Splunk, Elasticsearch, etc), there will be add
...
 
 ###### Output Buffer
 
-The `Output Buffer` editor allows you to describe how you want the output buffer to behave. You can find the documentation for the buffer fields [here](https://banzaicloud.com/docs/one-eye/logging-operator/configuration/plugins/outputs/buffer/).
+The `Output Buffer` editor allows you to describe how you want the output buffer to behave. You can find the documentation for the buffer fields [here](https://kube-logging.dev/docs/configuration/plugins/outputs/buffer/).
```
Comment (jillian-maroket) [inline docs/logging/harvester-logging.md:176]:
```suggestion
The editor allows you to describe the preferred output buffer behavior using various [fields](https://kube-logging.dev/docs/configuration/plugins/outputs/buffer/).
```

File: docs/logging/harvester-logging.md [hunk 6]
```diff
@@ -214,7 +214,7 @@ There must be at least one existing `ClusterOutput` or `Output` that can be atta
 
 ###### Filters
 
-Filters allow you to transform, process, and mutate the logs. In the text edit, you will find descriptions of the supported filters, but for more information, you can visit the list of [supported filters](https://banzaicloud.com/docs/one-eye/logging-operator/configuration/plugins/filters/).
+Filters allow you to transform, process, and mutate the logs. In the text edit, you will find descriptions of the supported filters, but for more information, you can visit the list of [supported filters](https://kube-logging.dev/docs/configuration/plugins/filters/).
```
Comment (jillian-maroket) [inline docs/logging/harvester-logging.md:217]:
```suggestion
Filters allow you to transform, process, and mutate the logs. For more information, see the list of supported [filters](https://kube-logging.dev/docs/configuration/plugins/filters/).
```

File: docs/logging/harvester-logging.md [hunk 7]
```diff
@@ -256,7 +256,7 @@ kubectl apply -f elasticsearch-logging.yaml
 
 ##### Referencing Secrets
 
-There are 3 ways Banzai Cloud allows specifying secret values via yaml values.
+There are 3 ways `Logging Operator` allows specifying secret values via yaml values.
```
Comment (jillian-maroket) [inline docs/logging/harvester-logging.md:259]:
```suggestion
You can define secret values (in YAML format) using any of the following methods:
```

File: docs/logging/harvester-logging.md [hunk 8]
```diff
@@ -285,7 +285,7 @@ tls_cert_path:
          key: <kubernetes-secret-key>
 ```
 
-For more information, you can find the related documentation [here](https://banzaicloud.com/docs/one-eye/logging-operator/configuration/plugins/outputs/secret/).
+For more information, you can find the related documentation [here](https://kube-logging.dev/docs/configuration/plugins/outputs/secret/).
```
Comment (jillian-maroket) [inline docs/logging/harvester-logging.md:288]:
```suggestion
For more information, see [Secret definition](https://kube-logging.dev/docs/configuration/plugins/outputs/secret/).
```

**Review by jillian-maroket — COMMENTED** [review]
I have reworded parts of the legacy text to improve readability. Some parts require checking for technical correctness, but most changes are straightforward language edits for better UX.

File: docs/logging/harvester-logging.md (hunk 3 above)
Comment (bk201) [reply docs/logging/harvester-logging.md:39]: Let's just update the non-EOL doc, starting from v1.3

**Comment by w13915984028** [comment]
@jillian-maroket Thanks for the rewording, I updated and please take a new look.

**Review by jillian-maroket — APPROVED** [review]

**Pull request merged**